tests/test_manager.py

```diff
@@ -55,40 +55,41 @@ def test_validate_failures(self):
         """
         Ensures publishing and finalizing report error properly.
         """
-        def test_nodes():
-            one_child_item = self.PublishItem("one_child", "one_child", "one_child")
+        one_child_item = self.PublishItem("one_child", "one_child", "one_child")
```

Comment:
moving these out of the `test_nodes` method allows to reuse them when validating the list returned by the validate method.